bHY6yh-de4661/cod-bxxqqq_bHY6yh-de4661.xml

```diff
@@ -0,0 +1,121 @@
+         <titleStmt>
+            <title>Rubrica 18</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+               <name>Grant Schreiber</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 18</title>
+               <date when="2023-06-01">June 1, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+              <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-01" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+           <pb ed="#B" n="8-r"/>
+           <cb ed="#B" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de4661">
+            <head xml:id="bHY6yh-de4661-Hd1e108">Rubrica 18</head>
+            <head xml:id="bHY6yh-de4661-Hd1e111" type="question-title">De iuramenti religione</head>
+            <p xml:id="bHY6yh-de4661-d1e114">
+              De iuramenti religione Rubrica 18
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e119">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de4661-Rd1e116">In leuitico capitulo 19</ref> <quote xml:id="bHY6yh-de4661-Qd1e125">non periurabis 
+              <lb ed="#B"/>in nomine meo nec pollues nomen 
+              <lb ed="#B"/>dei tui et cetera etc etc etc etc</quote>
+            </p>
+           <pb ed="#B" n="8-v"/>
+           <cb ed="#B" n="a"/>
+            <p xml:id="bHY6yh-de4661-d1e133">
```

Comment:
page breaks and columns breaks always need to be inside a `<p>` tag. See my adjustment below.

```suggestion
            </p>
            <p xml:id="bHY6yh-de4661-d1e133">
              <pb ed="#B" n="8-v"/>
              <cb ed="#B" n="a"/>

```